### ISSUE TITLE
I only get a 404

### ISSUE THREAD

**Issue by IeuanK** [photoprism/photoprism#3797]
#### 1. What is not working as documented?

PhotoPrism is not loading. The logs show no errors, but I get a black screen with this image:  

![image](https://github.com/photoprism/photoprism/assets/1075598/e2740930-d001-4561-9314-6eea4d558b17)

#### 2. How can we reproduce it?

I have no idea what is going wrong, so I can't provide steps to reproduce  

#### 3. What behavior do you expect?

App loading as usual

#### 4. What could be the cause of your problem?

I can't imagine any reason this would happen, at least not without errors

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

With debug mode turned on, the log only shows this:  

time="2023-10-07T14:44:29Z" level=debug msg="server: GET / (404) [3.715894ms]"
time="2023-10-07T14:44:29Z" level=debug msg="server: GET / (404) [2.260027ms]"
time="2023-10-07T14:44:32Z" level=debug msg="server: GET / (404) [2.463227ms]"
time="2023-10-07T14:44:36Z" level=debug msg="server: GET / (404) [2.292991ms]"
time="2023-10-07T14:44:37Z" level=debug msg="server: GET / (404) [2.122208ms]"
time="2023-10-07T14:44:40Z" level=debug msg="server: GET / (404) [2.167047ms]"
time="2023-10-07T14:45:06Z" level=debug msg="server: GET / (404) [2.036ms]"
time="2023-10-07T14:45:09Z" level=debug msg="server: GET / (404) [2.0515ms]"

#### 6. Which software versions do you use?

I use the latest docker tag, the docker host runs Debian.  

MariaDB version as per Adminer is `MySQL version: 5.5.5-10.11.3-MariaDB-1:10.11.3+maria~ubu2204`

I've tried various browsers, with and without extensions. I've tried accessing through NginxPM as well as directly accessing the port.

#### 7. On what kind of device is PhotoPrism installed?

(a) Device / Processor Type: Raspberry Pi 4, Intel Core i7-3770, AMD Ryzen 7 3800X,...  
No clue what CPU, but it's just a `Genuine Intel(R) CPU  @ 2.10GHz`. Other software meant for x64 processors runs fine.  
  
(b) Physical Memory & Swap Space in GB  
Plenty, over 180GB memory free  

(c) Storage Type: HDD, SSD, RAID, USB, Network Storage,...  
It's a ZFS pool with 10 drives, again no issue with other software, including software that sees heavy disk usage

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

I normally use a very simple proxy host within Nginx Proxy Manager, but accessing the container directly by exposing the port shows the same result

**Comment by lastzero** [comment]
see https://docs.photoprism.app/getting-started/troubleshooting/#app-not-loading

**Comment by IeuanK** [comment]
I already went through that checklist and double-checked everything.   
The issue is 100% on the container side, there is no reasonable explanation for it to explicitly serve a 404 if it were a basic JS issue, but I've confirmed that that also is not the case.  

None of the items on that checklist apply to my issue.  


Not sure why you're closing this, because it's not 'completed', it's still broken and the docs are no help in this case.
